web/public/mgmt/config.yml

```diff
@@ -2522,6 +2522,54 @@ collections:
           - { label: "school/course", value: "school-course" }
           - { label: "object or vehicle", value: "object-vehicle" }
 
+  - name: "raffle-bingo-steps"
+    label: "Raffle Bingo Steps (Navigator with Webflow mappings)"
+    folder: "content/src/roadmaps/raffle-bingo-steps"
+    summary: "{{displayname}} - {{name}}"
```

Comment:
[sand] We don't have a `displayname` for these items, so they're rendering with a leading dash.
```suggestion
    summary: "{{name}}"
```
<img width="679" alt="Screenshot 2024-09-30 at 11 58 57 AM" src="https://github.com/user-attachments/assets/be1ab94d-1d6f-48cc-be29-af0b5871eb74">